pipenv/exceptions.py

```diff
@@ -296,14 +291,11 @@ def __init__(self, message=None, **kwargs):
             message = "Failed to create virtual environment."
         self.message = message
         extra = kwargs.pop("extra", None)
-        if extra is not None and isinstance(extra, STRING_TYPES):
-            # note we need the format interpolation because ``crayons.ColoredString``
-            # is not an actual string type but is only a preparation for interpolation
-            # so replacement or parsing requires this step
-            extra = ANSI_REMOVAL_RE.sub("", f"{extra}")
+        if extra is not None and isinstance(extra, str):
```

Comment:
Well this is a big improvement!

Reply:
This is also working without checking for `not None`. It's also shorter.